### PR TITLE
Bug fix: Adding `chem_opt = 202` to `n2o5_hetchem` conditional

### PR DESCRIPTION
**Pull request by lukeconibear**
**TYPE:** bug fix

**KEYWORDS:** `chem_opt = 202`, `n2o5_hetchem = 1`

**SOURCE:** Luke Conibear (University of Leeds)

**DESCRIPTION OF CHANGES:**
_Problem:_
N2O5 heterogeneous chemistry without the Cl- pathway (`n2o5_hetchem = 1`) already works with default MOSAIC aerosol schemes (e.g. `chem_opt = 201/202/601/611`). However, `chem_opt = 202` was not included in the `config_flags%n2o5_hetchem == 1` conditional within `chemics_init.F`.

_Solution:_
Added `chem_opt = 202` to the `config_flags%n2o5_hetchem == 1` conditional within `chemics_init.F`.

**ISSUE:** None

**LIST OF MODIFIED FILES:**
```bash
M       chem/chemics_init.F
```

### CODE REVIEW

File: chem/chemics_init.F [hunk 1]
```diff
@@ -451,7 +451,7 @@ subroutine chem_init (id,chem,emis_ant,scalar,dt,bioemdt,photdt,chemdt,stepbioe,
             (config_flags%chem_opt >= 31  .AND. config_flags%chem_opt <= 34)  .OR. &
              config_flags%chem_opt == 170 .OR.  config_flags%chem_opt == 198  .OR. &
              config_flags%chem_opt == 199 .OR.  config_flags%chem_opt == 201  .OR. &
-             config_flags%chem_opt == 203 .OR.                                     &
+             config_flags%chem_opt == 203 .OR.  config_flags%chem_opt == 202  .OR. &
```
Comment (davegill) [inline chem/chemics_init.F:454]: @lukeconibear 
Luke,
I see that your mod minimizes differences. However, the rest of the chem_opt comparisons in this IF test are in numerical order. Maybe swap the location of the 202 and 203 values?

Comment (lukeconibear) [reply chem/chemics_init.F]: Sure, done.

**Comment by davegill** [comment]
jenkins:
```
Please find result of the WRF regression test cases in the attachment. This build is for Commit ID: cb4326e3cb48c39d9f5fe839a2f4ee54b4a892f9, requested by: lukeconibear for PR: https://github.com/wrf-model/WRF/pull/1368. For any query please send e-mail to David Gill.

    Test Type              | Expected  | Received |  Failed
    = = = = = = = = = = = = = = = = = = = = = = = =  = = = =
    Number of Tests        : 19           18
    Number of Builds       : 48           46
    Number of Simulations  : 166           164        0
    Number of Comparisons  : 105           104        0

    Failed Simulations are: 
    None
    Which comparisons are not bit-for-bit: 
    None
```

File: chem/chemics_init.F (hunk 1 above)
Comment (davegill) [reply chem/chemics_init.F]: Thanks

**Review by jordanschnell — APPROVED** [review]
Approved by Chem

**Pull request merged**